I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 417, in script
    call screen apartment(clean(store.apartment_data), store.game_state.time)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "renpy/common/00action_control.rpy", line 131, in __call__
    renpy.show_screen(self.screen, *self.args, **self.kwargs)
Exception: Screen set_performance is not known.


-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 417, in script
    call screen apartment(clean(store.apartment_data), store.game_state.time)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/ast.py", line 2259, in execute
    self.call("execute")
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/exports.py", line 3347, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/core.py", line 4543, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/screen.py", line 770, in event
    rv = self.child.event(ev, x, y, st)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1403, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 281, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 1403, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/layout.py", line 281, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/behavior.py", line 1142, in event
    return handle_click(self.clicked)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/behavior.py", line 1075, in handle_click
    rv = run(action)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/behavior.py", line 379, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_control.rpy", line 131, in __call__
    renpy.show_screen(self.screen, *self.args, **self.kwargs)
  File "/home/brent/dataville-renpy/renpy-8.1.3-sdk/renpy/display/screen.py", line 1220, in show_screen
    raise Exception("Screen %s is not known.\n" % (name[0],))
Exception: Screen set_performance is not known.


Linux-6.2.0-39-generic-x86_64-with-glibc2.35 x86_64
Ren'Py 8.1.3.23091805
DataVille 1.0
Tue Jan  2 17:19:38 2024
